simple-markdown.js

```diff
@@ -1289,7 +1289,7 @@ var defaultRules = {
         // We break on any symbol characters so that this grammar
         // is easy to extend without needing to modify this regex
         match: inlineRegex(
-            /^[\s\S]+?(?=[^0-9A-Za-z\s\u00ff-\uffff]|\n\n| {2,}\n|\w+:\S|$)/
+            /^[\s\S]+?(?=[^0-9A-Za-z\s\u00C0-\u00FF]|\n\n| {2,}\n|\w+:\S|$)/
```

Comment:
I believe this makes any unicode character (anything past \u00ff) end the text parsing, which we probably don't want, since they're unlikely to be delimiters for other rules (and aren't delimiters for any default rules).

Could we just change this to `\u00c0-\uffff` and get the same effect?